admin_manual/installation/source_installation.rst

```diff
@@ -22,20 +22,33 @@ Apache and MariaDB, using `the Nextcloud .tar archive
 * :ref:`php_fpm_tips_label`
 * :ref:`other_HTTP_servers_label`
 
-.. note:: Admins of SELinux-enabled distributions such as CentOS, Fedora, and 
-   Red Hat Enterprise Linux may need to set new rules to enable installing 
+.. note:: Admins of SELinux-enabled distributions such as CentOS, Fedora, and
+   Red Hat Enterprise Linux may need to set new rules to enable installing
    Nextcloud. See :ref:`selinux_tips_label` for a suggested configuration.
 
+.. _snaps_label:
+
+Installing via Snap Packages
+-------------------------------------
+
+A snap is a zip file containing an application together with its dependencies, and a description of how it should safely be run on your system, especially the different ways it should talk to other software. Most importantly snaps are designed to be secure, sandboxed, containerised applications isolated from the underlying system and from other applications. 
+    
+To install the Nextcloud Snap Package (available on Ubuntu 16.04 LTS and above), run the following command in a terminal::
```

Comment:
Maybe remove the mention of Ubuntu as snapd technology is available on different distros?